Rakefile

```diff
@@ -69,7 +69,8 @@ end
 
 desc 'Install packaging_rake_tasks gem package'
 task :install => :tarball do
-  sh 'sudo gem install --local package/packaging_rake_tasks*.gem'
+  sudo = (!Process.euid.zero? && File.exists?("/usr/bin/sudo")) ? "/usr/bin/sudo" : ""
+  sh "#{sudo} gem install --local package/packaging_rake_tasks*.gem"
```

Comment:
another option is to use `su -c` in case sudo is not available

Reply:
This `rake install` task is called only by developers when testing the gem locally, I'd keep it simple...